### PR TITLE
export content from msdn 4/10

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: xml/System.Data.Objects.DataClasses/RelatedEnd.xml [hunk 1]
```diff
@@ -823,12 +839,30 @@
...
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks  
+The `ValidateEntityForAttach<TEntity>` method is called by Object Services during an attach operation.
+
+]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">
+          When the object being attached is <see langword="null" />.
```
Comment (rpetrusha) [inline xml/System.Data.Objects.DataClasses/RelatedEnd.xml]: Exception information needs to be flush left; this is rendering as a code block.

File: xml/System.Data.Objects.DataClasses/RelatedEnd.xml [hunk 2]
```diff
@@ -853,12 +887,32 @@
...
+Object Services obtains the <xref:System.Data.Objects.ObjectQuery%601> by calling the [IRelatedEnd.CreateSourceQuery](xref:System.Data.Objects.DataClasses.RelatedEnd.System%23Data%23Objects%23DataClasses%23IRelatedEnd%23CreateSourceQuery) method.
+
+Object Services calls the `ValidateLoad<TEntity>` method before executing a load operation to ensure that the operation will succeed.
+
+]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">
+          When the source object was retrieved by using a <see cref="F:System.Data.Objects.MergeOption.NoTracking" /> query and the <see cref="T:System.Data.Objects.MergeOption" /> is not <see cref="F:System.Data.Objects.MergeOption.NoTracking" /> or the related objects are already loaded.
+
+          -or-
```
Comment (rpetrusha) [inline xml/System.Data.Objects.DataClasses/RelatedEnd.xml]: This is also rendering as a code block; it should be flush left.

File: xml/System.Data.Objects/CurrentValueRecord.xml [hunk 3]
```diff
@@ -257,10 +257,18 @@
...
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="i">The ordinal of the field.</param>
+        <summary>This method is not supported in this release.</summary>
+        <returns>The field value as a <see cref="T:System.Data.Common.DbDataReader" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks  
+This method is not supported in this release.
```
Comment (rpetrusha) [inline xml/System.Data.Objects/CurrentValueRecord.xml]: Perhaps just "This method is not supported."?

File: xml/System.Data.Objects/CurrentValueRecord.xml [hunk 4]
```diff
@@ -366,10 +374,18 @@
...
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="ordinal">The ordinal of the field.</param>
+        <summary>This method is not supported in this release.</summary>
+        <returns>The field value as a <see cref="T:System.Data.Common.DbDataReader" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks  
+This method is not supported in this release.
```
Comment (rpetrusha) [inline xml/System.Data.Objects/CurrentValueRecord.xml]: This method is not supported?

File: xml/System.Data.Objects/CurrentValueRecord.xml [hunk 5]
```diff
@@ -1311,10 +1327,18 @@
...
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="ordinal">The ordinal of the field.</param>
+        <summary>This method is not supported in this release.</summary>
+        <returns>The field value as an <see cref="T:System.Data.IDataReader" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks  
+This method is not supported in this release.
```
Comment (rpetrusha) [inline xml/System.Data.Objects/CurrentValueRecord.xml]: This method is not supported?

File: xml/System.Data/DataView.xml [hunk 6]
```diff
@@ -1870,7 +1870,7 @@
         <Parameter Name="value" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="value">A <see cref="System.Object" /> value.</param>
+        <param name="value">A <see cref="T:System.Object" /> value.</param>
```
Comment (rpetrusha) [inline xml/System.Data/DataView.xml]: nit: This and a number of other parameters are described as "A Object value" or "A Int32 value". "A" should be "An".

File: xml/System.Data.Objects/CurrentValueRecord.xml (hunk 3 above)
Comment (mairaw) [reply xml/System.Data.Objects/CurrentValueRecord.xml]: I've removed the remarks and just changed the summary to say that it throws the NSE in all cases.

**Pull request merged**